baby buddy upgrade failed

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

baby buddy

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/babybuddy.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

9.1.7

### 📝 Provide a clear and concise description of the issue.

I'm trying to upgrade an existing installation from 2.7.1 to 2.8.0 and the upgrade was unsuccessful. 


### 🔄 Steps to reproduce the issue.

run the community script in side existing baby buddy lxc

### ❌ Paste the full error output (if available).

`    ____        __             ____            __    __     
   / __ )____ _/ /_  __  __   / __ )__  ______/ /___/ /_  __
  / __  / __ `/ __ \/ / / /  / __  / / / / __  / __  / / / /
 / /_/ / /_/ / /_/ / /_/ /  / /_/ / /_/ / /_/ / /_/ / /_/ / 
/_____/\__,_/_.___/\__, /  /_____/\__,_/\__,_/\__,_/\__, /  
                  /____/                           /____/   

  ✔️   Update available: babybuddy 2.7.1 → 2.8.0
  ⏳   Upgrade uv from 0.8.11 to 0.11.7/dev/fd/63: line 8043: /etc/bash_completion.d/uv: No such file or directory
  ✔️   Setup uv 0.11.7
  ✔️   Services Stopped
  ✔️   Cleaned old files
  ✔️   Deployed: babybuddy (2.8.0)
  ⏳   Updating Baby BuddyChecked 44 packages in 114ms
Traceback (most recent call last):
  File "/opt/babybuddy/manage.py", line 26, in <module>
    execute_from_command_line(sys.argv)
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/core/management/base.py", line 405, in run_from_argv
    parser = self.create_parser(argv[0], argv[1])
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/core/management/base.py", line 368, in create_parser
    self.add_arguments(parser)
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/core/management/commands/migrate.py", line 50, in add_arguments
    choices=tuple(connections),
            ^^^^^^^^^^^^^^^^^^
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/utils/connection.py", line 73, in __iter__
    return iter(self.settings)
                ^^^^^^^^^^^^^
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/utils/functional.py", line 47, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
                                         ^^^^^^^^^^^^^^^^^^^
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/utils/connection.py", line 45, in settings
    self._settings = self.configure_settings(self._settings)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/db/utils.py", line 148, in configure_settings
    databases = super().configure_settings(databases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/utils/connection.py", line 50, in configure_settings
    settings = getattr(django_settings, self.settings_name)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/conf/__init__.py", line 81, in __getattr__
    self._setup(name)
  File "/opt/babybuddy/.venv/lib/python3.11/site-packages/django/conf/__init__.py", line 61, in _setup
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Requested setting DATABASES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
  ✖️   in line 51: exit code 1 (General error / Operation not permitted): while executing command  python manage.py migrate

  --- Last 20 lines of log ---
[2026-04-16 21:34:47] [INFO] Checking for update: babybuddy
[2026-04-16 21:34:47] [OK] Update available: babybuddy 2.7.1 → 2.8.0
[2026-04-16 21:34:48] [INFO] Upgrade uv from 0.8.11 to 0.11.7
[2026-04-16 21:34:49] [OK] Setup uv 0.11.7
[2026-04-16 21:34:49] [INFO] Stopping Services
[2026-04-16 21:34:50] [OK] Services Stopped
[2026-04-16 21:34:50] [INFO] Cleaning old files
[2026-04-16 21:34:50] [OK] Cleaned old files
[2026-04-16 21:34:51] [INFO] Fetching GitHub release: babybuddy (2.8.0)
[2026-04-16 21:34:52] [OK] Deployed: babybuddy (2.8.0)
[2026-04-16 21:34:52] [INFO] Updating Baby Buddy
[2026-04-16 21:34:53] [ERROR] in line 51: exit code 1 (General error / Operation not permitted): while executing command  python manage.py migrate
  -----------------------------------

 📋 Full log: /tmp/create-lxc-fe8bfbd8.log`

### 🖼️ Additional context (optional).

_No response_

--- a/ct/babybuddy.sh
+++ b/ct/babybuddy.sh
@@ -48,6 +48,7 @@ function update_script() {
     mv /tmp/production.py.bak /opt/babybuddy/babybuddy/settings/production.py
     source .venv/bin/activate
     $STD uv pip install -r requirements.txt
+    export DJANGO_SETTINGS_MODULE=babybuddy.settings.production
     $STD python manage.py migrate
     msg_ok "Updated ${APP}"
 

